src/org/zaproxy/zap/extension/exportReport/Export/ReportExport.java

```diff
@@ -0,0 +1,328 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.io.Writer;
+import java.nio.channels.Channels;
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.TransformerFactoryConfigurationError;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.extension.ViewDelegate;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.model.SiteMap;
+import org.parosproxy.paros.model.SiteNode;
+import org.zaproxy.zap.extension.exportReport.json.JSONException;
+import org.zaproxy.zap.extension.exportReport.json.JSONObject;
+import org.zaproxy.zap.extension.exportReport.json.XML;
+import org.zaproxy.zap.extension.exportReport.model.AlertItem;
+import org.zaproxy.zap.extension.exportReport.model.Alerts;
+import org.zaproxy.zap.extension.exportReport.model.Report;
+import org.zaproxy.zap.extension.exportReport.model.Sites;
+import org.zaproxy.zap.utils.XMLStringUtil;
+import org.zaproxy.zap.view.ScanPanel;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.ls.DOMImplementationLS;
+import org.w3c.dom.ls.LSSerializer;
+import org.xml.sax.SAXException;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ * CLASS		:	ReportExport.java 
+ * DESC			:	Retrieves all the data, parses it into a report type and returns it back to ExportReport.java.
+ * CREATED ON	:	MARCH 10TH, 2016
+ * CURRENT VER	:	V1.0
+ * SOURCE		:	https://github.com/JordanGS/workspace/tree/master/zap-extensions/src/org/zaproxy/zap/extension/exportReport
+ */
+
+/* 
+ * MODIFED BY	:	<NAME> - <GIT USER>
+ * MOD DATE		:	
+ * MOD VERSION	:	<VERSION OF PLUGIN>
+ * MOD DESC		:	
+ */
+
+public class ReportExport {
+
+	private static final String[] MSG_RISK = { "Informational", "Low", "Medium", "High" };
+	private static final String[] MSG_CONFIDENCE = { "False Positive", "Low", "Medium", "High", "Confirmed" };
+	
```

Comment:
Might be the right/wrong place, i found them at 

https://github.com/zaproxy/zaproxy/blob/develop/src/org/parosproxy/paros/core/scanner/Alert.java

I saw that the name changed in the past from MSG_RELIABILITY to MSG_CONFIDENCE, since there was no get method that returned the array i created my own local instance.  I thought that was the proper way to go forward but i can use the the one in the above linked Alert.java.